tools/labs/docker/kernel/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:18.04
+
+RUN apt-get update
+RUN apt-get install -y software-properties-common
+RUN apt-get install -y gcc-multilib
+RUN apt-get install -y libncurses5-dev
+RUN apt-get install -y bc
+RUN apt-get install -y qemu-system-x86
+RUN apt-get install -y qemu-system-arm
+RUN apt-get install -y python3
+RUN apt-get install -y minicom
+RUN apt-get install -y git
+RUN apt-get install -y wget
+RUN apt-get install -y curl
+RUN apt-get install -y sudo
+
+RUN apt-get install -y iproute2
+RUN apt-get install -y netcat-openbsd
+RUN apt-get install -y vim
+RUN apt-get install -y dnsmasq
+RUN apt-get install -y iputils-ping
+RUN apt-get install -y bash-completion
+RUN apt-get install -y build-essential
+RUN apt-get install -y bison
+RUN apt-get install -y flex
+
+RUN apt-get -y clean
+
+RUN rm -rf /var/lib/apt/lists/*
+
```

Comment:
We should also install packages to build and run 32 bit programs. Please check if the the ``device_drivers`` lab compiles.

Reply:
`gcc-multilib` and `libc6-i386` are usually required.

I had this issue in the lab:
```
/usr/bin/ld: skipping incompatible /usr/lib/gcc/x86_64-linux-gnu/8/libgcc.a when searching for -lgcc
```

It required installing `lib32gcc-8-dev`.

Reply:
I have added libc6-i386.

Reply:
I see you added `lib32gcc-8-dev`. Does that work for the `device_drivers` lab pointed out by @valighita ?

Reply:
It works with `gcc-multilib` package installed.